components/project-card/project-card.jsx

```diff
@@ -217,11 +300,11 @@ class ProjectCard extends React.Component {
   }
 
   renderDescription() {
-    return this.props.description.split(`\n`).map(paragraph => <p>{paragraph}</p>);
+    return this.props.description.split(`\n`).map((paragraph, i) => <p key={i}>{paragraph}</p>);
```

Comment:
Don't use list position as a key. Just use `key={paragraph}` so that reordering doesn't incur heavy DOM modification